lib/modules/hosts/vidgif.js

```diff
@@ -1,15 +1,15 @@
 export default {
-	moduleID: 'swirl',
-	name: 'swirl',
-	domains: ['swirl.xyz'],
+	moduleID: 'vidgif',
+	name: 'vidgif',
+	domains: ['vidgif.com'],
```

Comment:
Please keep the `swirl.xyz` domain in this array so old links still work (put it after `vidgif.com`)